extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -593,8 +603,10 @@ public SecurityIdentity apply(SecurityIdentity identity) {
                                     @Override
                                     public Throwable apply(Throwable tInner) {
                                         if (tInner instanceof AuthenticationRedirectException) {
+                                            LOG.debugf("Starting the final redirect");
                                             return tInner;
                                         }
+                                        LOG.debugf("ID token verification has failed: %s" + tInner.getMessage());
```

Comment:
```suggestion
                                        LOG.debugf("ID token verification has failed: %s",tInner.getMessage());
```

Reply:
@gsmet I'm trying my best - we can disable at any moment really, but then it will never be fixed as the only way to fix it is to keep tracing in CIs what might've happened. I think it is reasonable as long the test is not interfering a lot.

Reply:
@sberyozkin so first, this has been failing for a veeeery long time thus why I want to get rid of it. Second, you have the option to reproduce it in your fork without impacting everyone.

So as I said, I'm OK for one more week but it will have to go at the end of the week from the `main` branch.

Reply:
e@gsmet 
>so first, this has been failing for a veeeery long time thus why I want to get rid of it.

I'm not comfortable with this comment because it is not true, your 4 `e` are over dramatizing the problem, there is no need for it. It started failing for the last 2 weeks may be. Also I'm conscious of not affecting everyone and this is why I'm trying to address it. Agreed that it can be pursued further on private forks, feel free to disable at any moment

Reply:
@gsmet Moreover, I've seen this test failing only once across all the PR since Thursday when after I tried to improve it, which is about 3 days.